### ISSUE TITLE
Cleanup for cnx-recipes -> ce-styles

### ISSUE THREAD

**Issue by kswanson33** [openstax/cnx-recipes#4635]
Background: `cnx-recipes` is an ~old~ venerable repo that used to support easybake recipes (now retired) and could run an old version of the book generation pipeline using docker-compose. Now we only use it to host and build style files; `cookbook` now holds the recipes and `enki` takes care of book generation. Files related to legacy functions can be removed. Furthermore, it's been a while since dependencies have received TLC, so we can do that here too.

1. Update used dependencies & remove unused dependencies
2. Edit Dockerfile to stop installing stuff we don't need
3. Filter all files related to styles & style generation to a new repo named ... drumroll ... `ce-styles` :)

**Comment by ajluk** [comment]
@kswanson33 During working on clean up appendix styles I have noticed something strange - in the `_appendixglossary-shapes` file the only necessary imported file is the `_appendixglossary-components` file, although there are a title and list components present in the shape: 
![image](https://user-images.githubusercontent.com/55203598/167105570-65e672f4-c6a3-48ec-ab20-f61f5e622d10.png)
In the past we used to add imports of all components files we used in a particular shape in the shapes file (example from chapter intro shape): 
![image](https://user-images.githubusercontent.com/55203598/167106184-10568ec9-fcaf-4dd1-a27e-a904390c8fb7.png)
I don't know of this is relevant for this issue, but just wanted to let you know about it.

**Comment by kswanson33** [comment]
I don't think this should affect this issue ... everything in your screenshots looks like it's in the `styles` folder, and we're keeping everything in `styles` + everything that supports styles. (**Cleanup** in this epic is less about cleaning up the framework (that would be more for the styles refactor project) and more about updating dependencies & removing stuff we don't need anymore.)

**Comment by kswanson33** [comment]
## Files to keep:

1. Directories

![directories cnx-recipes.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/0a205465-e710-4329-80f8-37501561f20d)

From the `script` folder, we just keep:
- `run`
- `build-styles`
- `build-impacted-styles`

2. Top level files

![top level files cnx-recipes.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/c6281294-b6e3-4692-8a45-9a1eb31caa37)

Within `styles`, we can get rid of: 
- `styles/test` directory
- `styles/build/build.js`

## Packages

![package json cnx-recipes.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/f6b6826c-3ed9-4f4e-9daa-f5fecf139be9)

tl;dr the only NPM packages we're keeping are to support stylelint (linting for styles & sass)

**Comment by kswanson33** [comment]
ok to take out TEG files :fire:

**Comment by kswanson33** [comment]
![image.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/51aa538e-e04c-4748-b250-de67e1bfa20d)

these are the lines in package.json that can be taken out. the only thing we're still using is the styles linter

**Comment by lwtchu** [comment]
donezo
